crates/modelardb_compression/src/models/gorilla.rs

```diff
@@ -13,12 +13,12 @@
  * limitations under the License.
  */
 
-//! Implementation of the Gorilla model type which uses the lossless compression
-//! method for floating-point values proposed for the time series management
-//! system Gorilla in the [Gorilla paper]. As this compression method compresses
-//! the values of a time series segment using XOR and a variable length binary
-//! encoding, aggregates are computed by iterating over all values in the
-//! segment.
+//! Implementation of the Gorilla model type which uses the lossless compression method for
+//! floating-point values proposed for the time series management system Gorilla in the [Gorilla
+//! paper]. The compression method described in the paper has been extended with support for lossy
```

Comment:
This is not something that should change in this PR but I think we should make sure that we mention that `Gorilla` might be lossy in the documentation for the compression component. Potential users might always expect lossless compression when they hear `Gorilla` so it should be made very clear.

Reply:
Agreed, we should make it very clear that we used Gorilla's compression method as a baseline, but that we have extended with more functionality. According to `rg -i gorilla` we currently only mention `Gorilla` in the Rust files and I have tried to ensure that the comments make it clear that `Gorilla` is now lossy within an error bound.